### DISCUSSION TITLE
tinkerboard ubuntu ,

### DISCUSSION THREAD

**Discussion by spydergto** [johang/sd-card-images#63] [General]
Hello ,

at work , trying to build the image. im limited to a windows 10 install , have VB with ubuntu 20.04 on it for linux work. Ive tried to follow the steps written out , downloaded the files and zcat'ted the image sd-card together. tried 20.04 and 23, both do the same thing when installed to the tinkerboard they only go to uboot once uboot hands off it goes no input on monitor.

Ive also tried making the image moving it to usb stick moving that to win10 desktop and using etcher to write sd card , they do the exact same thing. both start uboot , when it hands off nothing. no input display . any ideas ?

TY

Ive now tried using my pi3b+ at home to make the image and xfr it to work via google drive, and also the tinkerboard here at work with a diffrent sd card setup with debain and gotten the exact same results uboot boots then nothing no input 

at a loss

**Comment by johang** [comment]
These images are very minimal. You shouldn't expect display to work out of the box.

**Comment by spydergto** [comment]
Awesome , TY for taking the time to let me know. basically i wrote the image as you described. put the sd into my tinkerboard and booted it with HDMI plugged in and got a blank / black screen . im at work so the network is huge and i didnt think to go scanning the entire network for it . i got no usual boot up screen at all where linux runs through its list of booted options . only a black screen which is why i was figuring i did something wrong. 

I will try again, how should i access the device then , I've not got my USB to TTYL cable at work that's at home for serial USB shell , if the image booted correctly , I'm guessing SSH is setup over Ethernet ?
